README.md

```diff
@@ -57,7 +60,7 @@ docker pull dedis/drand
 ### Via Golang
 
 1. Make sure that you have a working [Golang installation](https://golang.org/doc/install) and that your [GOPATH](https://golang.org/doc/code.html#GOPATH) is set.
-2. **TODO: install the DFINITY crypto library?**
+2. Install the [pairing-based crypto library](https://github.com/dfinity/bn). **TODO: more details needed?**
```

Comment:
The details are in the BN's README. although not much detailed I admit. Anyway, our first users will probably use the docker thing, and we'll need to package the BN library better in the future.